health/health.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package health
```

Comment:
I think this package could move under `admin/health` since it is self contained now

Reply:
Fixed!  I typically decouple service from api, but I agree that there's not a lot of value in this case.